### ISSUE TITLE
Error message: "system error detected"

### ISSUE THREAD

**Issue by rreiner** [ntop/ntopng#8234]
**Environment**:
* OS name: [e.g. Ubuntu]

Raspberry Pi OS

* OS version: [e.g. 18.04]

Debian Bookworm

`Linux pinkeye 6.1.0-rpi8-rpi-v8 #1 SMP PREEMPT Debian 1:6.1.73-1+rpt1 (2024-01-25) aarch64 GNU/Linux`

* Architecture: [e.g. amd64]

aarch64

* ntopng version/revision: [e.g. 5.1.211223]

 6.1.240219

**What happened**:

Received an Emerg alert:

```
[20/02/2024 00:02:00]  [System] [System Error] [system]  System error detected in ntopng, please report it. Error message: /usr/share/ntopng/scripts/lua/modules/blog_utils.lua:172: attempt to call a nil value (global 'split')
```

**How did you reproduce it?**

Occurred during normal operation

**Debug Information**:

```
Feb 19 00:01:20 pinkeye ntopng[786]: 19/Feb/2024 00:01:20 [fetch_blog_feed.lua:15] [blog_utils.lua:146] Fetching latest ntop blog posts...
Feb 19 00:02:08 pinkeye ntopng[786]: 19/Feb/2024 00:02:08 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'Abuse.ch URLhaus' [https://urlhaus.abuse.ch/downloads/hostfile/]... OK
Feb 19 00:02:09 pinkeye ntopng[786]: 19/Feb/2024 00:02:09 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'Emerging Threats' [https://rules.emergingthreats.net/fwrules/emerging-Block-IPs.txt]... OK
Feb 19 00:02:09 pinkeye ntopng[786]: 19/Feb/2024 00:02:09 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'IPsum Threat Intelligence Feed' [https://raw.githubusercontent.com/stamparm/ipsum/master/ipsum.txt]... OK
Feb 19 00:02:10 pinkeye ntopng[786]: 19/Feb/2024 00:02:10 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'NoCoin Filter List' [https://raw.githubusercontent.com/hoshsadiq/adblock-nocoin-list/master/hosts.txt]... OK
Feb 19 00:02:10 pinkeye ntopng[786]: 19/Feb/2024 00:02:10 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'SSLBL Botnet C2 IP Blacklist' [https://sslbl.abuse.ch/blacklist/sslipblacklist.txt]... OK
Feb 19 00:02:11 pinkeye ntopng[786]: 19/Feb/2024 00:02:11 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'Stratosphere Lab' [https://mcfp.felk.cvut.cz/publicDatasets/CTU-AIPP-BlackList/Todays-Blacklists/AIP_historical_blacklist_prioritized_by_newest_attackers.csv]... OK
Feb 19 00:02:12 pinkeye ntopng[786]: 19/Feb/2024 00:02:12 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'ThreatFox' [https://threatfox.abuse.ch/downloads/hostfile/]... OK
Feb 19 00:02:12 pinkeye ntopng[786]: 19/Feb/2024 00:02:12 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'dshield 7 days' [https://raw.githubusercontent.com/firehol/blocklist-ipsets/master/dshield_7d.netset]... OK
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded Abuse.ch URLhaus: 214 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded Emerging Threats: 1049 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded IPsum Threat Intelligence Feed: 37588 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded NoCoin Filter List: 409 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded SSLBL Botnet C2 IP Blacklist: 38 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded Stratosphere Lab: 10697 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded ThreatFox: 29494 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:646] Loaded dshield 7 days: 29 rules
Feb 19 00:02:13 pinkeye ntopng[786]: 19/Feb/2024 00:02:13 [housekeeping.lua:36] [lists_utils.lua:767] Loaded Category Lists (30117 hosts, 49401 IPs, 0 JA3) loaded in 0 sec
Feb 20 00:01:29 pinkeye ntopng[786]: 20/Feb/2024 00:01:29 [fetch_blog_feed.lua:15] [blog_utils.lua:146] Fetching latest ntop blog posts...
Feb 20 00:01:30 pinkeye ntopng[786]: 20/Feb/2024 00:01:30 [LuaEngine.cpp:697] WARNING: /usr/share/ntopng/scripts/lua/modules/blog_utils.lua:172: attempt to call a nil value (global 'split') [/usr/share/ntopng/scripts/callbacks/daily-delayed/system/fetch_blog_feed.lua]
Feb 20 00:01:30 pinkeye ntopng[786]: 20/Feb/2024 00:01:30 [LuaEngine.cpp:697] WARNING: /usr/share/ntopng/scripts/lua/modules/blog_utils.lua:172: attempt to call a nil value (global 'split') [/usr/share/ntopng/scripts/callbacks/daily-delayed/system/fetch_blog_feed.lua]
Feb 20 00:02:16 pinkeye ntopng[786]: 20/Feb/2024 00:02:16 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'Abuse.ch URLhaus' [https://urlhaus.abuse.ch/downloads/hostfile/]... OK
Feb 20 00:02:16 pinkeye ntopng[786]: 20/Feb/2024 00:02:16 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'Emerging Threats' [https://rules.emergingthreats.net/fwrules/emerging-Block-IPs.txt]... OK
Feb 20 00:02:17 pinkeye ntopng[786]: 20/Feb/2024 00:02:17 [housekeeping.lua:36] [lists_utils.lua:444] Updating list 'IPsum Threat Intelligence Feed' [https://raw.githubusercontent.com/stamparm/ipsum/master/ipsum.txt]..
```

**Comment by NicoMaio** [comment]
Hi @rreiner,
the issue should be fixed by 14d9dd5bb401540aca89a8318a78c54775a6da9e

Please update and let us know if it's alright now

**Comment by MatteoBiscosi** [comment]
Verified as fixed, please reopen if needed
